docs/QSG/build_from_scratch.md

```diff
@@ -2,296 +2,103 @@
-
-		graphviz \\
-
-		hwloc-devel \\
-
-		ipmctl \\
-
-		java-1.8.0-openjdk \\
-
-		json-c-devel \\
-
-		lcov \\
-
-		libaio-devel \\
-
-		libcmocka-devel \\
-
-		libevent-devel \\
-
-		libipmctl-devel \\
-
-		libiscsi-devel \\
-
-		libtool \\
-
-		libtool-ltdl-devel \\
-
-		libunwind-devel \\
-
-		libuuid-devel \\
-
-		libyaml-devel \\
-
-		Lmod \\
-
-		lz4-devel \\
-
-		make \\
-
-		man \\
-
-		maven \\
-
-		nasm \\
-
-		ndctl \\
-
-		numactl-devel \\
-
-		openssl-devel \\
-
-		pandoc \\
-
-		patch \\
-
-		patchelf \\
-
-		pciutils \\
-
-		python3-pip \\
-
-		python36-Cython \\
-
-		python36-devel \\
-
-		python36-distro \\
-
-		python36-jira \\
-
-		python36-numpy \\
-
-		python36-paramiko \\
+Download DAOS source code using the following command:
 
-		python36-pylint \\
+The DAOS repository is hosted on [GitHub](https://github.com/daos-stack/daos).
 
-		python36-requests \\
+To checkout the 2.0 version, simply run:
 
-		python36-requests \\
+```
+$ git clone --recurse-submodules -b release/2.0 https://github.com/daos-stack/daos.git
+$ cd daos
+```
 
-		python36-tabulate \\
+## Install Prerequisites
 
-		python36-pyxattr \\
+To build DAOS and its dependencies, several software packages must be installed
+on the system. This includes scons, libuuid, cmocka, ipmctl, and several other
+packages usually available on all the Linux distributions. Moreover, a Go
+version of at least 1.10 is required.
 
-		python36-scons \\
+Some DAOS tests use MPI. The DAOS build process uses the environment modules
+package to detect the presence of MPI. If none is found, the build will skip
+building those tests.
 
-		sg3\_utils \\
+Scripts to install all the required packages are provided for each supported
+distribution.
 
-		valgrind-devel \\
-
-		yasm && \\
-
-		yum clean all
-
-		yum -y install openmpi3-devel && yum clean all
-
-### openSUSE Build Installation Prerequisites
-
-		**git clone**
-
-		\# Run as root
-
-		zypper \--non-interactive update && \\
-
-		zypper \--non-interactive install \\
-
-		boost-devel \\
-
-		clang \\
-
-		cmake \\
-
-		cunit-devel \\
-
-		curl \\
-
-		doxygen \\
-
-		flex \\
-
-		fuse3-devel \\
-
-		gcc \\
-
-		gcc-c++ \\
-
-		git \\
-
-		graphviz \\
-
-		gzip \\
-
-		hwloc-devel \\
-
-		java-1\_8\_0-openjdk-devel \\
-
-		libaio-devel \\
-
-		libcmocka-devel \\
-
-		libevent-devel \\
-
-		libiscsi-devel \\
-
-		libjson-c-devel \\
-
-		libltdl7 \\
-
-		liblz4-devel \\
-
-		libnuma-devel \\
-
-		libopenssl-devel \\
-
-		libtool \\
-
-		libunwind-devel \\
-
-		libuuid-devel \\
-
-		libyaml-devel \\
-
-		lua-lmod \\
-
-		make \\
-
-		man \\
-
-		maven \\
-
-		nasm \\
-
-		openmpi3-devel \\
-
-		pandoc \\
-
-		patch \\
-
-		patchelf \\
-
-		python3-devel \\
-
-		python3-distro \\
-
-		python3-pip \\
-
-		python3-pyxattr \\
-
-		python3-PyYAML \\
-
-		python3-tabulate \\
+### CentOS
 
-		scons \\
+On CentOS7, please run the following command from the DAOS tree.
 
-		sg3\_utils \\
+```bash
+$ ./utils/scripts/install-centos7.sh
```

Comment:
Should be called with sudo or the commands inside the bash script.

Reply:
Fixed in new version.